core/embed/rust/src/micropython/func.rs

```diff
@@ -16,3 +18,35 @@ impl Func {
 
 // SAFETY: We are in a single-threaded environment.
 unsafe impl Sync for Func {}
+
+pub type FuncVar = ffi::mp_obj_fun_builtin_var_t;
+
+impl FuncVar {
+    /// Convert variable argument "static const" function to a MicroPython
+    /// object.
+    pub const fn as_obj(&'static self) -> Obj {
+        // SAFETY:
+        //  - We are an object struct with a base and a type.
+        //  - 'static lifetime holds us in place.
+        //  - MicroPython is smart enough not to mutate `mp_obj_fun_builtin_var_t`
+        //    objects.
+        unsafe { Obj::from_ptr(self as *const _ as *mut _) }
+    }
+}
+
+// SAFETY: We are in a single-threaded environment.
+unsafe impl Sync for FuncVar {}
+
+/// Unpack arguments for variable argument function.
+pub fn unpack_args<'a>(n_args: usize, args: *const Obj) -> &'a [Obj] {
```

Comment:
This definitely needs to be `unsafe fn`.

Reply:
Good point. I've removed it entirely in db922be161c0da87925763260c351983fe676f52, as the helpers in `src/util.rs` already do this and more.